package.json

```diff
@@ -34,17 +34,20 @@
     "@angular/core": "^16.2.10",
     "@angular/forms": "^16.2.10",
     "@angular/material": "^16.2.9",
-    "angular-material-expansion-panel": "^0.7.2",
     "@angular/platform-browser": "^16.2.10",
     "@angular/platform-browser-dynamic": "^16.2.10",
     "@angular/router": "^16.2.10",
+    "@fortawesome/fontawesome-free": "^6.5.1",
+    "angular-material-expansion-panel": "^0.7.2",
     "chart.js": "^4.4.0",
     "chartjs-adapter-date-fns": "^3.0.0",
     "date-fns": "^2.30.0",
+    "material-design-icons": "^3.0.1",
     "moment": "^2.29.4",
     "ngx-spinner": "^16.0.2",
     "rxjs": "~7.8.1",
     "tslib": "^2.6.2",
+    "typeface-roboto": "^1.1.13",
```

Comment:
the pacakge typeface-roboto has been deprecated and we should move to fontsource-roboto as per the [readme](https://classic.yarnpkg.com/en/package/typeface-roboto#readme)

Reply:
Thanks, I will update the PR